packages/bpk-component-chip-group/src/BpkChipGroup.module.scss

```diff
@@ -30,6 +30,7 @@
   display: flex;
   margin: 0;
   padding: 0;
+  align-items: baseline;
```

Comment:
The label doesn't fill the height (like chips do) so needs alignment to be properly positioned

Reply:
Can you add the built css file too?

Reply:
Added 👍